engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/PerTestHelper.scala

```diff
@@ -137,6 +135,7 @@ private[ejea] class PerTestHelper(implicit val system: ActorSystem) extends Mock
       ioActor = ioActorProbe.ref,
       jobStoreActor = jobStoreProbe.ref,
       callCacheReadActor = callCacheReadActorProbe.ref,
+      callCacheWriteActor = callCacheWriteActorProbe.ref,
```

Comment:
Actually I think I'm going to punt on this.  All the previous logic was doing was testing the creation of the actor, which is now done up in the CromwellRootActor.